seq2science/envs/cite-seq-count.yaml

```diff
@@ -0,0 +1,5 @@
+name: cite-seq-count
+channels:
+  - guillaumecharbonnier
+dependencies: 
+  - guillaumecharbonnier::cite-seq-count=1.4.3 

```

Comment:
there is a newer version on the bioconda channel: https://anaconda.org/search?q=cite-seq-count

Reply:
bumped version to 1.4.4